corehq/apps/consumption/shortcuts.py

```diff
@@ -50,7 +51,7 @@ def get_default_consumption(domain, product_id, location_type, case_id):
     consumption = get_default_monthly_consumption(domain, product_id, location_type, case_id)
 
     if consumption:
-        return consumption / Decimal(DAYS_IN_MONTH)
+        return consumption / DAYS_IN_MONTH
```

Comment:
Unless `consumption` is a `Decimal`, this will now return a `float` instead of a `Decimal`, which has subtly different characteristics. Do you know for sure that returning a `float` here is fine? Same below.

Reply:
You're right, I don't know for sure. https://github.com/dimagi/commcare-hq/pull/19096/commits/f1a53fda236f4a10641c0ef49e7390113f1fb3fe